fsspec/tests/test_spec.py

```diff
@@ -647,6 +649,12 @@ def _open(
             None,
             DummyTestFS.get_test_paths() + [DummyTestFS.root_marker],
         ),
+        [
+            (Path("misc/foo.txt"),),
+            False,
+            None,
+            ["misc/foo.txt" if platform.system() != "Windows" else "misc\\foo.txt"],
```

Comment:
```suggestion
            [f"misc{os.sep}foo.txt"],
```
maybe?

...but shouldn't we always be returning posix paths always?

Reply:
I don't know all of the details, but I assumed that a `LocalFileSystem` on Windows would not be happy with POSIX-only paths.